ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__vboulos-acm-ocp4.12-lp-interop.yaml

```diff
@@ -0,0 +1,74 @@
+base_images:
+  acmqe-grc-test:
+    name: "2.7"
+    namespace: acm-qe
+    tag: acmqe-grc-test
+  application-ui-test:
+    name: "2.7"
+    namespace: acm-qe
+    tag: application-ui-test
+  clc-ui-e2e:
+    name: "2.7"
+    namespace: acm-qe
+    tag: clc-ui-e2e
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  fetch-managed-clusters:
+    name: autotest
+    namespace: acm-qe
+    tag: fetch-managed-clusters
+  observability-core-automation:
+    name: "2.7"
+    namespace: acm-qe
+    tag: observability-core-automation
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: acm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      OPERATORS: |
+        [
+          {"name": "advanced-cluster-management", "source": "redhat-operators", "channel": "release-2.7", "install_namespace": "ocm", "target_namespaces": "ocm"},
+          {"name": "ansible-automation-platform-operator", "source": "redhat-operators", "channel": "stable-2.3-cluster-scoped", "install_namespace": "openshift-operators", "target_namespaces": "", "operator_group": "global-operators"},
+          {"name": "compliance-operator", "source": "redhat-operators", "channel": "release-0.1", "install_namespace": "openshift-compliance", "target_namespaces": "openshift-compliance"},
+          {"name": "gatekeeper-operator-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators", "target_namespaces": "", "operator_group": "global-operators"},
+          {"name": "openshift-gitops-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators", "target_namespaces": "", "operator_group": "global-operators"},
+          {"name": "oadp-operator", "source": "community-operators", "channel": "stable", "install_namespace": "openshift-adp", "target_namespaces": "openshift-adp"},
+          {"name": "container-security-operator", "source": "redhat-operators", "channel": "stable-3.6", "install_namespace": "openshift-operators", "target_namespaces": "", "operator_group": "global-operators"}
+        ]
+      ZONES_COUNT: "4"
+    post:
+    - ref: acm-tests-clc-destroy
+    - ref: gather-aws-console
+    - chain: ipi-deprovision
+    pre:
+    - chain: ipi-conf-aws
+    - chain: ipi-install
+    test:
+    - ref: install-operators
+    - ref: acm-mch
+    - ref: acm-tests-clc-create
+    - ref: acm-fetch-managed-clusters
+    - ref: acm-tests-obs
```

Comment:
We discussed removing this file and just merging the changes for the test refs.

If you do want to also merge this file I think we should just make sure `acm-tests-obs` is the last ref that we run.